expanded chart interims (#673)
SimpleDataLabsInc/prophecy-docs: docs/analysts/development/data-explorer/charts/charts.md

---
title: Charts
id: charts
slug: /analysts/charts
description: Charts you can create in your data samples
tags:
  - charts
---

> To view your data in a chart, switch to the **Visualization View** in the project header.

Reviewer: Just in case someone arrives at this page from google search or something:

```suggestion
Visualize interim data samples in your pipeline with charts. To view your data in a chart, open a data sample and switch to the **Visualization** tab of the [Data Explorer](/analysts/data-explorer).
```

![VisualizationView](./img/visualization-view.png)

> ## Chart Types

Reviewer: This section is super helpful as an overview!

```suggestion
## Chart types
```

> You can create the following charts in the **Visualization View**:

Reviewer: Suggestion: what do you think about styling it like **Visualization** view? I have only been bolding words that directly match the UI

Author: Not against it at all! I only did it this way because I saw many instances of Code View that way. I'll update it :)

| [Bar](/analysts/bar-chart)                 | Group your data by categories with rectangular bars with heights or lengths proportional to the values that they represent. |
| [Line](/analysts/line-chart)               | Visually represent your data over time or along a continuous range.                                                         |
| [Area](/analysts/area-chart)               | Line chart with filled areas under the lines to represent quantitative data over time or categories.                        |
| [Pie](/analysts/pie-chart)                 | Circular graph that divides into slices, where the arc length of each slice is proportional to the quantity it represents.  |
| [Candlestick](/analysts/candlestick-chart) | Financial chart that displays how prices change for an asset over time, such as stocks and currency.                        |
| [Map](/analysts/map-chart)                 | Uses a map to show how data is distributed across a geographic region.                                                      |

> | [Scatter](/analysts/scatter-chart)         | Uses Cartesian coordinates to show the relationship between two variables.                                                  |

Reviewer: Maybe have the scatter description be more simple? "Dots or bubbles mapped to coordinates that show the relationship between two variables" or something

## Filter fields

You can apply and remove filters to your data.

![FilterChart](./img/filter-chart.png)

### Apply a filter

For each filter you want to apply to your data:

1. Click **Filter** at the top of the chart.

1. Click **Add Filter**.

1. Configure your filter.

   Select the **Condition**, **Column**, and **Value** to filter for.

1. Click **Apply**.

### Remove a filter

For each filter you want to remove from your data:

1. Click **Filter** at the top of the chart.

1. Find the filter you want to remove.

1. Click the **trash can icon** at the top right corner.

## Add to business app

> 

Reviewer: Perhaps add a leading sentence, like

```suggestion
If you create a chart in the Data Explorer that would be useful in a Prophecy App, you can add it to an app template using the **Add to App** option. This way, end users can view the chart you created to visualize the data generated when they run the app. 
```

Also we are calling business apps `Prophecy Apps` in the doc 😅 Even though we keep saying business apps internally

Reviewer: Also, I tested it out, and if you already have a config of an app, it adds the chart to it. It doesn't even let you choose which config to add it to... just the first one. So it's really confusing. I'm going to bring it up in docs sync today

![AddChartToApp](./img/add-chart-to-app.png)

> For each filter you want to remove from your data:

Reviewer: I think this intro sentence is a vestige of a copy-paste 🙂

Author: haha you are right! will update with correct text 😅

1. Click **Add to App** at the top of the chart.

1. Select the business app you want to add it to.

>    After you select the app, Prophecy opens you app in a new window.

Reviewer: ```suggestion
   After you select the app, Prophecy opens your app in a new window.
```
